tests/MightNotMakeAssertions.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
this shim can probably be fully removed and replaced with calls to `$this->expectNotToPerformAssertions()`

Reply:
:+1: Done.